collect_app/src/main/java/org/odk/collect/android/activities/InstanceChooserList.java

```diff
@@ -68,7 +68,7 @@ public void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.chooser_list_layout);
 
         String order;
-        if (getIntent().getStringExtra(ApplicationConstants.BundleKeys.FORM_MODE).equalsIgnoreCase(ApplicationConstants.FormModes.EDIT_SAVED)) {
+        if ((ApplicationConstants.FormModes.EDIT_SAVED).equalsIgnoreCase(getIntent().getStringExtra(ApplicationConstants.BundleKeys.FORM_MODE))) {
```

Comment:
This is an unrelated change so please remove it from this pull request! 

Interestingly, I think it's the same kind of thing as the null check above -- we tend to put the unchanging thing after the changing one in tests but it looks like you have a different style. There's nothing wrong with the way you do it but for consistency it would be good to stick to one convention. 😊